exercise10_2.py

```diff
@@ -41,13 +41,13 @@
     print('File cannot be opened:', fname)
     quit()
 
-for line in fhand:
-    words = line.split()
-    if len(words) < 2 or words[0] != 'From':
+for l in fhand:
```

Comment:
I try to avoid single character names for variables, especially for code that is intended for third-party use. If it's code only I'll read, I might use `l` here. Even thought probably. I typically only use this convention if its something like, `for l in lines:` where it's completely obvious what `l` is.

In fact, the PEP8 conventions are pretty explicit about never using `l` as a variable name, https://www.python.org/dev/peps/pep-0008/#names-to-avoid